### PR TITLE
[IMP] MRP: Update Manufacturing step docs for Shop Floor

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR replaces the tablet view workflow in the One, Two, and Three-step manufacturing docs with a new section detailing the workflow for the Shop Floor module.

### CODE REVIEW

**Comment by samueljlieber** [comment]
Rebased to latest in https://github.com/odoo/documentation/commit/4efc6d2cd7aa82999cc1e83984ce6c2744519983

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 1]
```diff
@@ -81,32 +83,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
-:guilabel:`Produce All` button on the order.
+.. note::
+   The *Register Production* step appears on every work order card. It must be completed for the
+   first work order that is processed. After doing so, the step appears as already completed for
+   each remaining work order in the |MO|.
+
+After completing all of the steps for a work order, a button appears on the footer of the work order
+card. If any other work orders must be completed before the |MO| can be closed, the button is titled
+*Mark as Done*. If there are no additional work orders to complete, the button is titled *Close
+Production*.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:138]: I'm not sure when we use italics, but my known example is italicizing terms we're defining for the first time, like [specified in this doc](https://docs.google.com/document/d/1laiWC9ow0IjXPuQp5h8L_Tj98HbE8ymGn9v-FFOwlYM/edit#heading=h.90nibixh8llr)

If you're talking bout the text, "Mark as Done," maybe surround it with quotes? but i don't see why *not* to use a guilabel here to refer to the button itself, directly. What do you think? :smile: 
```suggestion
card. The :guilabel:`Mark as Done` button appears when any other work orders must be completed to
close the |MO|. If there are no additional work orders to complete, the :guilabel:`Close Production`
button appears, instead.
```

Also, optional rewording suggestion to use more active voice! feel free to ignore

**Review by Felicious — APPROVED** [review]
Great job with updating these manufacturing docs with the newest Shop Floor workflow, @hojo-odoo !

It's clear and easy to follow, and I like how you explained the behavior of the pop-ups and steps so thoroughly :blush: 

I just reviewed the Shop Floor workflow for the one-step manufacturing doc, like you suggested. Let me know if I missed anything!

I just had a few minor nitpicks about formatting. Feel free to contest these suggestions, especially if you think I'm wrong. I want to be aligned on the guilabel, etc usage as a team :smile:

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 2]
```diff
@@ -81,32 +83,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
-Next, click on the :guilabel:`Work Orders` tab, and then select the :guilabel:`📱 (tablet)` button
-on the line of the first work order to be processed. This opens the tablet view.
+.. image:: one_step_manufacturing/shop-floor-button.png
+   :align: center
+   :alt: The Open Shop Floor button for a work order on a manufacturing order.
+
+When accessed directly from a specific work order within an |MO|, *Shop Floor* defaults to the page
+for the work center where the work order is configured to be carried out. The page shows a card for
+the work order that displays the |MO| number, the product and number of units to be produced, and
+the steps required to complete the work order.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:107]:
```suggestion
the work order that displays the |MO| number, the product and number of units to be produced, along
with the steps required to complete the work order.
```

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 3]
```diff
@@ -81,32 +83,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+A work order is processed by completing each step listed on its card. This can be done by clicking
+on a step, and following the instructions listed on the pop-up window that appears. Once the step is
+completed, click :guilabel:`Next` to move on to the next step, if any are required.
 
-After opening the tablet view, Odoo *Manufacturing* automatically starts a timer that keeps track of
-how long the work order takes to complete. After completing the work order, click the
-:guilabel:`Mark as Done` button in the top-right corner of the tablet view.
+Alternatively, work order steps can be completed by clicking the checkbox that appears on the right
+side of the step's line on the work order card. When using this method, the step is marked as
+completed automatically, without a pop-up window appearing.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:119]:
```suggestion
Alternatively, work order steps can be completed by clicking the checkbox that appears on the right
side of the specific step's line on the work order card. When using this method, the step is marked
as completed automatically, without a pop-up window appearing.
```

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 4]
```diff
@@ -81,32 +83,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+inventory.
+
+After clicking :guilabel:`Mark as Done` or :guilabel:`Close Production`, each button is replaced by
+an :guilabel:`Undo` button. Click the :guilabel:`Undo` button before the work order card fades away
+to keep the work order open.
+
+.. tip::
+   This section details the basic workflow for processing an |MO| in the Shop Floor module. For a
+   more in-depth explanation of the module and all of its features, please see the :ref:`Shop Floor
+   overview <manufacturing/shop_floor/shop_floor_overview>` documentation.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:157]:
```suggestion
   more in-depth explanation of the module, and all of its features, please see the :ref:`Shop Floor
   overview <manufacturing/shop_floor/shop_floor_overview>` documentation.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 5]
```diff
@@ -106,35 +106,78 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
-Tablet view workflow
---------------------
+Shop Floor workflow
+-------------------
 
-To complete the work orders for an |MO| using the tablet view, begin by navigating to
-:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, and then select a
-manufacturing order.
+To complete the work orders for an |MO| using the *Shop Floor* module, begin by navigating to
+:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, and then select an |MO|.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:113]:
```suggestion
:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, then select an |MO|.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 6]
```diff
@@ -106,35 +106,78 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
 
-Next, click on the :guilabel:`Work Orders` tab, and then select the :guilabel:`📱 (tablet)` button
-on the line of the first work order to be processed. This opens the tablet view.
+On the |MO|, click on the :guilabel:`Work Orders` tab, and then select the :guilabel:`↗️ (square
+with arrow coming out of it)` button on the line of the first work order to be processed. Doing so
+opens a :guilabel:`Work Orders` pop-up window, with details and processing options for the work
+order.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:118]:
```suggestion
On the |MO|, click on the :guilabel:`Work Orders` tab, and select the :guilabel:`↗️ (square with
arrow coming out of it)` button on the line of the first work order to be processed. Doing so
opens a :guilabel:`Work Orders` pop-up window, with details and processing options for the work
order.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 7]
```diff
@@ -106,35 +106,78 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+open the *Shop Floor* module.
+
+.. image:: three_step_manufacturing/shop-floor-button.png
+   :align: center
+   :alt: The Open Shop Floor button for a work order on a manufacturing order.
+
+When accessed directly from a specific work order within an |MO|, *Shop Floor* defaults to the page
+for the work center where the work order is configured to be carried out. The page shows a card for
+the work order that displays the |MO| number, the product and number of units to be produced, and
+the steps required to complete the work order.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:130]:
```suggestion
the work order that displays the |MO| number, the product and number of units to be produced, along
with the steps required to complete the work order.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 8]
```diff
@@ -106,35 +106,78 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
-   :alt: The tablet view button for a work order on a manufacturing order.
+   :alt: A work order card on a work center page in the Shop Floor module.
+
+A work order is processed by completing each step listed on its card. This can be done by clicking
+on a step, and following the instructions listed on the pop-up window that appears. Once the step is
+completed, click :guilabel:`Next` to move on to the next step, if any are required.
+
+Alternatively, work order steps can be completed by clicking the checkbox that appears on the right
+side of the step's line on the work order card. When using this method, the step is marked as
+completed automatically, without a pop-up window appearing.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:142]:
```suggestion
side of the specific step's line on the work order card. When using this method, the step is marked
as completed automatically, without a pop-up window appearing.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 9]
```diff
@@ -106,35 +106,78 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+inventory.
+
+After clicking :guilabel:`Mark as Done` or :guilabel:`Close Production`, each button is replaced by
+an :guilabel:`Undo` button. Click the :guilabel:`Undo` button before the work order card fades away
+to keep the work order open.
+
+.. tip::
+   This section details the basic workflow for processing an |MO| in the Shop Floor module. For a
+   more in-depth explanation of the module and all of its features, please see the :ref:`Shop Floor
+   overview <manufacturing/shop_floor/shop_floor_overview>` documentation.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:180]:
```suggestion
   more in-depth explanation of the module, and all of its features, please see the :ref:`Shop Floor
   overview <manufacturing/shop_floor/shop_floor_overview>` documentation.
```

File: content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst [hunk 10]
```diff
@@ -103,31 +103,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
-To complete the work orders for an |MO| using the tablet view, begin by navigating to
+To complete the work orders for an |MO| using the *Shop Floor* module, begin by navigating to
 :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, and then select an |MO|.
 
-Next, click on the :guilabel:`Work Orders` tab, then select the :guilabel:`📱 (tablet)` button on
-the line of the first work order to be processed. This opens the tablet view.
+On the |MO|, click on the :guilabel:`Work Orders` tab, and then select the :guilabel:`↗️ (square
+with arrow coming out of it)` button on the line of the first work order to be processed. Doing so
+opens a :guilabel:`Work Orders` pop-up window, with details and processing options for the work
+order.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst:115]:
```suggestion
On the |MO|, click on the :guilabel:`Work Orders` tab, and select the :guilabel:`↗️ (square with
arrow coming out of it)` button on the line of the first work order to be processed. Doing so opens
a :guilabel:`Work Orders` pop-up window, with details and processing options for the work order.
```

File: content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst [hunk 11]
```diff
@@ -103,31 +103,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+open the *Shop Floor* module.
+
+.. image:: two_step_manufacturing/shop-floor-button.png
+   :align: center
+   :alt: The Open Shop Floor button for a work order on a manufacturing order.
+
+When accessed directly from a specific work order within an |MO|, *Shop Floor* defaults to the page
+for the work center where the work order is configured to be carried out. The page shows a card for
+the work order that displays the |MO| number, the product and number of units to be produced, and
+the steps required to complete the work order.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst:127]:
```suggestion
the work order that displays the |MO| number, the product and number of units to be produced, along
with the steps required to complete the work order.
```

File: content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst [hunk 12]
```diff
@@ -103,31 +103,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
-   :alt: The tablet view button for a work order on a manufacturing order.
+   :alt: A work order card on a work center page in the Shop Floor module.
+
+A work order is processed by completing each step listed on its card. This can be done by clicking
+on a step, and following the instructions listed on the pop-up window that appears. Once the step is
+completed, click :guilabel:`Next` to move on to the next step, if any are required.
+
+Alternatively, work order steps can be completed by clicking the checkbox that appears on the right
+side of the step's line on the work order card. When using this method, the step is marked as
+completed automatically, without a pop-up window appearing.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst:139]:
```suggestion
side of the specific step's line on the work order card. When using this method, the step is marked
as completed automatically, without a pop-up window appearing.
```

File: content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst [hunk 13]
```diff
@@ -103,31 +103,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+The final step on a work order card is titled *Register Production*. This step is used to register
+the number of product units that were produced. If the number produced is equal to the number that
+the |MO| was created for, click the :guilabel:`# Units` button on the right side of the line to
+automatically register that number as the quantity produced.
 
-Clicking :guilabel:`Mark as Done` while there is at least one more work order left to complete opens
-a page that lists the next work order. Click on that work order to open it in the tablet view.
+If a different number must be entered, click the :guilabel:`Register Production` step to open a
+pop-up window. Enter the number of units produced in the :guilabel:`Units` field, and then click
+:guilabel:`Validate` to register that number.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst:148]:
```suggestion
pop-up window. Enter the number of units produced in the :guilabel:`Units` field, then click
:guilabel:`Validate` to register that number.
```

File: content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst [hunk 14]
```diff
@@ -103,31 +103,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
-:guilabel:`Mark as Done`. In this case, the |MO| can be closed at a later time by clicking the
-:guilabel:`Produce All` button on the order.
+After completing all of the steps for a work order, a button appears on the footer of the work order
+card. If any other work orders must be completed before the |MO| can be closed, the button is titled
+:guilabel:`Mark as Done`. If there are no additional work orders to complete, the button is titled
+:guilabel:`Close Production`.
+
+Clicking :guilabel:`Mark as Done` causes the work order card to fade away. Once it disappears
+completely, the work order's status is marked as *Finished* on the |MO|, and the next work order
+appears in the Shop Floor module, on the page of the work center where it is configured to be
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst:162]:
```suggestion
appears in the *Shop Floor* module, on the page of the work center where it is configured to be
```

File: content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst [hunk 15]
```diff
@@ -103,31 +103,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+inventory.
+
+After clicking :guilabel:`Mark as Done` or :guilabel:`Close Production`, each button is replaced by
+an :guilabel:`Undo` button. Click the :guilabel:`Undo` button before the work order card fades away
+to keep the work order open.
+
+.. tip::
+   This section details the basic workflow for processing an |MO| in the Shop Floor module. For a
+   more in-depth explanation of the module and all of its features, please see the :ref:`Shop Floor
+   overview <manufacturing/shop_floor/shop_floor_overview>` documentation.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst:177]:
```suggestion
   This section details the basic workflow for processing an |MO| in the *Shop Floor* module. For a
   more in-depth explanation of the module, and all of its features, please see the :ref:`Shop Floor
   overview <manufacturing/shop_floor/shop_floor_overview>` documentation.
```

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 16]
```diff
@@ -81,32 +83,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+   each remaining work order in the |MO|.
+
+After completing all of the steps for a work order, a button appears on the footer of the work order
+card. If any other work orders must be completed before the |MO| can be closed, the button is titled
+:guilabel:`Mark as Done`. If there are no additional work orders to complete, the button is titled
+:guilabel:`Close Production`.
+
+Clicking :guilabel:`Mark as Done` causes the work order card to fade away. Once it disappears
+completely, the work order's status is marked as *Finished* on the |MO|, and the next work order
+appears in the Shop Floor module, on the page of the work center where it is configured to be
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:142]:
```suggestion
appears in the *Shop Floor* module, on the page of the work center where it is configured to be
```

File: content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst [hunk 17]
```diff
@@ -81,32 +83,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+Clicking :guilabel:`Close Production` causes the work order card to fade away. Once it disappears,
+the |MO| is marked as *Done*, and the units of the product that were produced are entered into
+inventory.
+
+After clicking :guilabel:`Mark as Done` or :guilabel:`Close Production`, each button is replaced by
+an :guilabel:`Undo` button. Click the :guilabel:`Undo` button before the work order card fades away
+to keep the work order open.
+
+.. tip::
+   This section details the basic workflow for processing an |MO| in the Shop Floor module. For a
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst:155]:
```suggestion
   This section details the basic workflow for processing an |MO| in the *Shop Floor* module. For a
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 18]
```diff
@@ -106,35 +106,78 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
-:guilabel:`Mark as Done`. In this case, the |MO| can be closed at a later time by clicking the
-:guilabel:`Produce All` button on the |MO|.
+After completing all of the steps for a work order, a button appears on the footer of the work order
+card. If any other work orders must be completed before the |MO| can be closed, the button is titled
+:guilabel:`Mark as Done`. If there are no additional work orders to complete, the button is titled
+:guilabel:`Close Production`.
+
+Clicking :guilabel:`Mark as Done` causes the work order card to fade away. Once it disappears
+completely, the work order's status is marked as *Finished* on the |MO|, and the next work order
+appears in the Shop Floor module, on the page of the work center where it is configured to be
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:165]:
```suggestion
appears in the *Shop Floor* module, on the page of the work center where it is configured to be
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 19]
```diff
@@ -106,35 +106,78 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+Clicking :guilabel:`Close Production` causes the work order card to fade away. Once it disappears,
+the |MO| is marked as *Done*, and the units of the product that were produced are entered into
+inventory.
+
+After clicking :guilabel:`Mark as Done` or :guilabel:`Close Production`, each button is replaced by
+an :guilabel:`Undo` button. Click the :guilabel:`Undo` button before the work order card fades away
+to keep the work order open.
+
+.. tip::
+   This section details the basic workflow for processing an |MO| in the Shop Floor module. For a
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:178]:
```suggestion
   This section details the basic workflow for processing an |MO| in the *Shop Floor* module. For a
```

**Review by ksc-odoo — APPROVED** [review]
@hojo-odoo Just finished my Peer Review. Awesome work (as always) 👍 Once these minor comments are addressed, this should be ready for the next stage!

**Comment by StraubCreative** [comment]
Hi @ksc-odoo please approve or request changes here and I can do final review, thanks!

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst (hunk 8 above)
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:142]:
```suggestion
side of the step's line on the work order card. When using this method, the step is automatically
marked as completed, without a pop-up window appearing.
```

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 20]
```diff
@@ -106,35 +106,78 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+:guilabel:`Validate` to register that number.
 
-Once the final work order for the |MO| has been reached, a :guilabel:`Mark as Done and Close MO`
-button appears on the tablet view in addition to the :guilabel:`Mark as Done` button. Click
-:guilabel:`Mark as Done and Close MO` to mark the |MO| as :guilabel:`Done`, and register the
-manufactured product(s) into inventory.
+.. note::
+   The *Register Production* step appears on every work order card. It must be completed for the
+   first work order that is processed. After doing so, the step appears as already completed for
+   each remaining work order in the |MO|.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:156]:
```suggestion
   The *Register Production* step appears on every work order card. It **must** be completed for the
   first work order that is processed. After doing so, the step appears as already completed for
   each remaining work order in the |MO|.
```

File: content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst [hunk 21]
```diff
@@ -103,31 +103,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
-   :alt: The tablet view button for a work order on a manufacturing order.
+   :alt: A work order card on a work center page in the Shop Floor module.
+
+A work order is processed by completing each step listed on its card. This can be done by clicking
+on a step and following the instructions listed on the pop-up window that appears. Once the step is
+completed, click :guilabel:`Next` to move on to the next step, if any are required.
+
+Alternatively, work order steps can be completed by clicking the checkbox that appears on the right
+side of the step's line on the work order card. When using this method, the step is marked as
+completed automatically, without a pop-up window appearing.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst:139]:
```suggestion
side of the step's line on the work order card. When using this method, the step is automatically
marked as completed, without a pop-up window appearing.
```

File: content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst [hunk 22]
```diff
@@ -103,31 +103,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
+If a different number must be entered, click the :guilabel:`Register Production` step to open a
+pop-up window. Enter the number of units produced in the :guilabel:`Units` field, and then click
+:guilabel:`Validate` to register that number.
 
-Once the final work order for the |MO| has been reached, a :guilabel:`Mark as Done and Close MO`
-button appears on the tablet view in addition to the :guilabel:`Mark as Done` button. Click
-:guilabel:`Mark as Done and Close MO` to mark the |MO| as :guilabel:`Done`, and register the
-manufactured product(s) into inventory.
+.. note::
+   The *Register Production* step appears on every work order card. It must be completed for the
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst:151]:
```suggestion
   The *Register Production* step appears on every work order card. It **must** be completed for the
```

**Review by ksc-odoo — APPROVED** [review]
@hojo-odoo Great work on this doc. Just a few, super-minor suggestions that need your attention. Once those are done, I think this one can be passed along to @StraubCreative

File: content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst [hunk 23]
```diff
@@ -106,35 +106,78 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
-   :alt: The tablet view button for a work order on a manufacturing order.
+   :alt: A work order card on a work center page in the Shop Floor module.
+
+A work order is processed by completing each step listed on its card. This can be done by clicking
+on a step, and following the instructions listed on the pop-up window that appears. Once the step is
+completed, click :guilabel:`Next` to move on to the next step, if any are required.
+
+Alternatively, work order steps can be completed by clicking the checkbox that appears on the right
+side of the step's line on the work order card. When using this method, the step is automatically
+marked as completed, without a pop-up window appearing.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst:142]: Add a space in "check box"
```suggestion
Alternatively, work order steps can be completed by clicking the check box that appears on the
right side of the step's line on the work order card. When using this method, the step is
automatically marked as completed, without a pop-up window appearing.
```

File: content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst [hunk 24]
```diff
@@ -103,31 +103,75 @@ same process for each work order listed on the :guilabel:`Work Orders` tab.
...
-   :alt: The tablet view button for a work order on a manufacturing order.
+   :alt: A work order card on a work center page in the Shop Floor module.
+
+A work order is processed by completing each step listed on its card. This can be done by clicking
+on a step and following the instructions listed on the pop-up window that appears. Once the step is
+completed, click :guilabel:`Next` to move on to the next step, if any are required.
+
+Alternatively, work order steps can be completed by clicking the checkbox that appears on the right
+side of the step's line on the work order card. When using this method, the step is automatically
+marked as completed, without a pop-up window appearing.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst:139]:
```suggestion
Alternatively, work order steps can be completed by clicking the check box that appears on the
right side of the step's line on the work order card. When using this method, the step is
automatically marked as completed, without a pop-up window appearing.
```

**Review by samueljlieber — COMMENTED** [review]
Hi @hojo-odoo 👋 this is a great PR! I only have a small change on each of the modified files, please see below.

Edit: I misunderstood a previous comment from ZST on "checkbox" vs "check box". Please ignore my comments regarding this. I am updating the Vale linter :)

Also, I think the [IMP] commit tag would be more fitting for this PR since no docs are being added. 

Tag me once more after making these edits and I can move this PR forward. Thank you!

---

Please note there a few uncompressed images listed below that must have been missed in a previous PR. Can you please make a [FIX] pr to compress these all of these in another PR targeting the base branch (`16.0` I believe) where these images were introduced? 
```
one_step_manufacturing/start-button.png
two_step_manufacturing/manufacturing-type.png
two_step_manufacturing/start-button.png
two_step_manufacturing/mo-bread-crumb.png
three_step_manufacturing/manufacturing-type.png
three_step_manufacturing/mo-bread-crumb.png
```
> Reminder you can run `make review` on an image directory to check for compression :)

**Comment by hojo-odoo** [comment]
@StraubCreative This is ready for merge.

**Review by samueljlieber — APPROVED** [review]
Looks good to me, thank you @hojo-odoo 👍

**Comment by StraubCreative** [comment]
[b461287](https://github.com/odoo/documentation/pull/6747/commits/b461287b25f83d66200d2b08ad8b9b7f4489b2b3): corrected commit tag.

**Review by StraubCreative — APPROVED** [review]
Look good, thanks @hojo-odoo 

To consider: 
- adjust docs and screenshots in `17.0` as well since the **Open Shop Floor** button is no longer present. 
- talking to RD or PO regarding the button being missing (was this a feature or a bug?)

@robodoo r+

**Pull request closed without merging**